qiskit/visualization/matplotlib.py

```diff
@@ -785,8 +785,8 @@ def _draw_ops(self, verbose=False):
                 elif len(q_xy) == 1:
                     disp = op.name
                     if param:
-                        prm = '({})'.format(param)
-                        if len(prm) < 20:
+                        prm = str(param)
+                        if len(prm) < 28:
```

Comment:
I don't actually think this will work as we want it to. The number 20 was not arbitrarily picked it was done because the length of the box for single qubit gates with parameters with a subtext is only drawn wide enough for a parameter list that is 20 characters long. If we want to remove this restriction (which I think we do), What we need to do here is actually change https://github.com/Qiskit/qiskit-terra/blob/master/qiskit/visualization/matplotlib.py#L228 and https://github.com/Qiskit/qiskit-terra/blob/master/qiskit/visualization/matplotlib.py#L641 to make the box size and spacing dynamic based on a subtext longer than the gate name. Once we do that we can just drop this if else block and always call `self._gate()` with the subtext regardless of length.

Right now if you pass in a wide parameter list you can get a parameter list that goes outside the box like the u3 in this example I ran with this change:

![Screenshot_2019-08-15_17-35-41](https://user-images.githubusercontent.com/2447371/63128521-1ffff580-bf83-11e9-9c31-d23e91f0628d.png)